src/game/bettercamera.inc.h

```diff
@@ -344,18 +360,23 @@ static void newcam_rotate_button(void)
                 }
             }
             else
-                newcam_yaw_acc = newcam_adjust_value(newcam_yaw_acc,(-gPlayer2Controller->stickX/4));
-        }
+            {
+                newcam_yaw_acc = newcam_adjust_value(newcam_yaw_acc,(-gPlayer2Controller->stickX/8, intendedXMag));
+            }
```

Comment:
`error: expected '}' before 'else'`

Reply:
Try now and see if this commit fixes this

Reply:
Alright. Resolved for good.